src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -1284,7 +1284,7 @@ public ModelBase GetModelForElement(XmlElement modelData)
                 string.Format("Unhandled model type: {0}", helper.ReadString("type", modelData.Name)));
         }
 
-        internal ModelBase GetModelInternal(Guid modelGuid)
+        public ModelBase GetModel(Guid modelGuid)
```

Comment:
Hi @marimano, please can you rename this back to `GetModelInternal`? It was named `GetModelInternal` not because of it being an `internal` method. It was made that way to signify that it is the *method that does the real work* (of getting a `ModelBase`). This will reduce the code changes here, thanks!